CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+### Release 2.2.11 (May 28, 2020)
+[Milestone#51](https://github.com/awslabs/amazon-kinesis-client/milestone/51)
+* Adjusting HTTP2 initial window size to 512 KB
+  * [PR#706](https://github.com/awslabs/amazon-kinesis-client/pull/706)
+* Updating protobuf-java to version 3.11.4
+  * [PR#718](https://github.com/awslabs/amazon-kinesis-client/pull/718)
+* Updating the AWS Java SDK to version 2.13.25
```

Comment:
May be we can add the fixes we are interested in

SDK bug fixes to be pulled in

Netty NIO HTTP Client

Bugfixes

    Fix an issue where the Netty client was prematurely considering an HTTP/2 request body as sent, but was still in the process of being transferred to the remote endpoint.

Bugfixes
    Expand Http2 connection-level flow control window when a new stream is acquired on that connection so that the connection-level window size is proportional to the number of streams.

Reply:
I'll check with doc writers on the wording for these and send a followup PR if it gets their approval. Will merge in the meantime to get started with release process